velox/substrait/SubstraitUtils.h

```diff
@@ -83,7 +91,9 @@ class SubstraitParser {
   /// words. Key: the Substrait function key word, Value: the Velox function key
   /// word. For those functions with different names in Substrait and Velox,
   /// a mapping relation should be added here.
-  std::unordered_map<std::string, std::string> substraitVeloxFunctionMap;
+  std::unordered_map<std::string, std::string> substraitVeloxFunctionMap_ = {
+      {"add", "plus"},
```

Comment:
Mapping functions is tricky as one needs to ensure that semantics match. For example, SQL + is expected to check for overflow. What are the semantics of the Substrait functions?

Reply:
The Substrait function is in the format of "function name: function signature". The function signature specifies the input types, and the optional or required enumerations. As below example shows, I think the checkoverflow can be set in the opt or req. Here the function map only maps the function name from Substrait to Velox, but the function signatures are not handled.
```
add(optional enumeration, i8, i8) => add:opt_i8_i8
extract(required enumeration, timestamp) => extract:req_ts

```